indexer/Cargo.toml

```diff
@@ -30,6 +30,7 @@ near-client = { git = "https://github.com/near/nearcore", rev = "4fce209e78fd100
 near-o11y = { git = "https://github.com/near/nearcore", rev = "4fce209e78fd1008c3a779101d080fe1570bbfe9" }
 near-client-primitives = { git = "https://github.com/near/nearcore", rev = "4fce209e78fd1008c3a779101d080fe1570bbfe9" }
 borsh = { version = "1.0.0", features = ["derive", "rc"] }
+serde_yaml = "0.9.34"
```

Comment:
NEAR codebase is using JSON, but the AVS configurations are all YAML. Main reason why I did that, otherwise the operator would fill a JSON and a YAML config.

Reply:
> Main reason why I did that, otherwise the operator would fill a JSON and a YAML config.

Well, that happens to be stack of NEAR, they anyway will get a json config as an output, keys, account as json outputs.

There's no point of bringing YAML into indexer since its already uses JSON. Moreover it will deserialize yaml and the serialize into JSON to write. 

Indexer is indexer, can live outside of SFFL and out operators. It is written on stack that uses JSON don't see any reason of bloating exe even more with unnecessary library

Reply:
Again, the point in bringing YAML is because this is specifically ths config the user inputs for the indexer. It's not an internal nearcore config, and I'd argue even if this specific config was already deserialized as a JSON we should support YAML as well (which is not uncommon) - see #67, it isn't nice having both a JSON and a YAML. I think not using a YAML is worse for consistency.